config/versions.yaml

```diff
@@ -33,6 +33,13 @@ installers:
       - 'centos7'
       - 'debian8'
 
+  - foreman: '1.16'
+    katello: '3.5'
+    puppet: 4
+    boxes:
+      - 'centos7'
+      - 'debian8'
```

Comment:
I think Debian 9 is also supported. @mmoll?

Reply:
yes. 🌀

Reply:
Good catch! Updated